doc/user/content/ops/troubleshooting.md

```diff
@@ -10,58 +10,120 @@ aliases:
 ---
 
 You can use the queries below for spot debugging, but it may also be
-helpful to make them more permanent tools by creating them as views for `SUBSCRIBE`ing or materialized views to read more efficiently. Note that the
-existence of these additional views may itself affect performance.
+helpful to monitor their output by `SUBSCRIBE`ing to their change stream, e.g.,
+by issuing a command `COPY (SUBSCRIBE (<query>) TO stdout`.
+Note that this additional monitoring may, however, itself affect performance.
+
+### Limitations on Introspection Sources and Views for Troubleshooting
+
+Importantly, the introspection sources and views used in the queries below
+should be considered subject to changes in the future without further notice.
+This is why these sources and views are all created in the `mz_internal`
+schema, and it is not possible to create higher-level views depending on them.
+
+Introspection sources are maintained by independently collecting internal logging
+information within each of the replicas of a cluster. Thus, in a multi-replica
+cluster, the queries below need to be directed to a specific replica by issuing
+the command `SET cluster_replica = <replica_name>`. Materialize also directly
```

Comment:
Maybe add a note that this enforces a single replica target for all `SELECT` queries, whether they read from introspection sources or not. This can be cancelled by issuing `RESET cluster_replica`.

Reply:
Done.